src/components/customerCases/customerCase/sections/text/TextSection.tsx

```diff
@@ -17,18 +17,26 @@ export default function TextSection({ section }: TextSectionProps) {
     <div className={styles.wrapper}>
       <div
         className={
-          styles.content +
-          (section.textBlockType === "framed" ? ` ${styles.highlighted}` : "")
+          styles.content /* +
```

Comment:
```suggestion
          styles.content
```

Reply:
ahh never mind! I guess this is for the frame/highlighted issue?

Reply:
Yes! I noted in the description why it was commented out and that it can be removed or added back in at a later date so we have the history and reasoning available for later!